src/app/locales/it.json

```diff
@@ -0,0 +1,819 @@
+{
+    "g": {
+        "age": "età",
+        "amount": "Quantità",
+        "and": "e",
+        "are_you_sure": "Sei sicuro?",
+        "ask": "Ask",
+        "back": "Indietro",
+        "balance": "Saldo",
+        "balances": "Saldi",
+        "bid": "Bid",
+        "blog": "Blog",
+        "browse": "Naviga",
+        "buy": "Acquista",
+        "buy_VESTING_TOKEN": "Acquista%(VESTING_TOKEN)s",
+        "buy_or_sell": "Acquista o vendi",
+        "by": "da",
+        "cancel": "Cancella",
+        "change_password": "Cambia Password",
+        "choose_language": "Scegli la lingua",
+        "clear": "Pulisci",
+        "close": "Chiudi",
+        "collapse_or_expand": "Riduci/Espandi",
+        "comments": "Commenti",
+        "confirm": "Conferma",
+        "convert": "Converti",
+        "date": "Data",
+        "delete": "Elimina",
+        "dismiss": "Abbandona",
+        "edit": "Modifica",
+        "email": "Email",
+        "feed": "Feed",
+        "feeds": "Feeds",
+        "follow": "Segui",
+        "for": "per",
+        "from": "da",
+        "go_back": "Indietro",
+        "hide": "Nascondi",
+        "in": "in",
+        "in_reply_to": "in risposta a",
+        "insufficient_balance": "Credito insufficiente",
+        "invalid_amount": "Importo errato",
+        "joined": "Connesso",
+        "loading": "Caricamento",
+        "login": "Login",
+        "logout": "Logout",
+        "memo": "Memo",
+        "more": "più",
+        "mute": "Silenzia",
+        "new": "Nuovo",
+        "newer": "Più recente",
+        "next": "Prossimo",
+        "no": "No",
+        "nothing_yet": "Ancora niente",
+        "ok": "Ok",
+        "older": "Meno recente",
+        "or": "o",
+        "order_placed": "Ordine piazzato",
+        "password": "Password",
+        "payouts": "Pagamenti",
+        "permissions": "Permessi",
+        "post": "Post",
+        "post_as": "Pubblica come",
+        "posts": "Posts",
+        "powered_up_100": "Powered Up 100%%",
```

Comment:
What's the guidance on translation of branded terminology? Are we leaving it as-is?